catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -459,6 +459,9 @@ const WorkDetails: FunctionComponent<Props> = ({ work, itemUrl }: Props) => {
                 imagesColor: null,
                 color: null,
                 locationsLicense: null,
+                languages: [],
+                subjectsLabel: null,
+                genresLabel: null,
```

Comment:
Why have these been added to the images route? Is this the issue where the works / images routes are coupled?

Reply:
Agreed, but the workRouteProps is shared between works and images and complains if taken off etc... I think that maybe a bigger refactor as the more filters we start added this will be a issue as we discussed.